app/src/main/res/xml/preferences.xml

```diff
@@ -37,6 +37,13 @@
             android:defaultValue="true"
             android:key="pref_show_backdrop"
             android:title="@string/lbl_show_backdrop" />
+        <ListPreference
+            android:defaultValue="VERTICAL"
```

Comment:
You missed changing the default here :wink:

Reply:
```suggestion
            android:defaultValue="HORIZONTAL"
```